haskell-stack/README.md

```diff
@@ -0,0 +1,28 @@
+# Haskell-stack dockerfile
+
+Creates an image with [`stack`](http://docs.haskellstack.org/en/stable/README/) installed.
+
+## Build
+
+To build, run:
+
+```
+cd haskell-stack
+docker build -t advancedtelematic/haskell-stack:1.0.2 .
+```
+
+## Usage
+
+This needs `stack setup` to be run and persisted between calls, so first up run the following:
+
+```
+docker create --name stack-cache -v /root/.stack tianon/true /bin/true 2>/dev/null || true'
+```
+
+Then use that volume for the rest of the calls like so:
+
+```
+docker run --net=host -v "$(pwd):/src" --volumes-from stack-cache advancedtelematic/haskell-stack stack
```

Comment:
I was using it for the to run `ota-plus-test-driver`, so that it could see everything on localhost, but that's been moved to the ota-plus-client repo. So here for just using `stack` as a command, shouldn't be necessary. I'll remove it.